libs/include/Thread.h

```diff
@@ -1,20 +0,0 @@
-/** Copyright (C) 2016 European Spallation Source */
-
-#include <thread>
-
-/** Wrapper class for pthread creation. Also 'locks' pthread to
- * specified lcore.
- *
- */
-class Thread : public std::thread {
-
```

Comment:
Removal of Thread.h and Thread.cpp was not mentioned in the pull request. I am guessing that they are not being used anymore? Can you clarify?

Reply:
Yes, no longer used anywhere